tesseract_core/sdk/engine.py

```diff
@@ -277,40 +287,53 @@ def build_image(
     Returns:
         An Image object representing an image on the local machine.
     """
+    if not isinstance(src_dir, Path):
+        src_dir = Path(src_dir)
+    if not isinstance(build_dir, Path):
+        build_dir = Path(build_dir)
+
     dockerfile_path = build_dir / "Dockerfile"
     with open(dockerfile_path, "w") as f:
         logger.debug(f"Writing Dockerfile to {dockerfile_path}")
         f.write(dockerfile)
 
     copytree(src_dir, build_dir / "__tesseract_source__")
 
-    if (Path(src_dir) / "tesseract_requirements.txt").exists():
-        local_dependencies, remote_dependencies = parse_requirements(
-            Path(src_dir) / "tesseract_requirements.txt"
-        )
-    else:
-        remote_dependencies = []
-        local_dependencies = []
-
-    if local_dependencies:
-        local_requirements_path = build_dir / "local_requirements"
-        Path.mkdir(local_requirements_path)
-        for dependency in local_dependencies:
-            src = Path(src_dir) / dependency
-            dest = build_dir / "local_requirements" / src.name
-            if src.is_file():
-                copy(src, dest)
-            else:
-                copytree(src, dest)
-
-    # We need to write a new requirements file in the build dir, where we explicitly
-    # removed the local dependencies
-    requirements_file_path = (
-        Path(build_dir) / "__tesseract_source__" / "tesseract_requirements.txt"
+    from tesseract_core import sdk
+
+    template_dir = Path(sdk.__file__).parent / "templates"
+
+    copy(
+        template_dir / requirements._build_script,
+        build_dir / "__tesseract_source__" / requirements._build_script,
     )
-    with requirements_file_path.open("w", encoding="utf-8") as f:
-        for dependency in remote_dependencies:
-            f.write(f"{dependency}\n")
+
+    if requirements.provider == "python-pip":
```

Comment:
What happens for conda + local dependencies?

Reply:
Is that even supported for conda?

Reply:
I guess they could have a section like this:

```yaml
- pip:
    - ./mypackage.whl
```

For now I'd be fine with not supporting that?